partiql-logical-planner/src/lower.rs

```diff
@@ -916,10 +917,18 @@ impl<'ast> Visitor<'ast> for AstToLogical {
         let name = _call.func_name.value.to_lowercase();
 
         if let Some(call_def) = self.fnsym_tab.lookup(name.as_str()) {
-            self.push_vexpr(call_def.lookup(&env));
+            match call_def.lookup(&env, name) {
+                Ok(lookup) => self.push_vexpr(lookup),
+                Err(err) => {
+                    // Include as error but allow lowering to proceed for multiple error reporting
+                    self.errors.push(err);
+                    self.push_vexpr(ValueExpr::Lit(Box::new(Value::Missing)))
```

Comment:
Push some `ValueExpr` on the stack (`Missing` literal) otherwise some later code that expects something to be on the `vexpr` stack will give a somewhat irrelevant error.